Assets/MixedRealityToolkit.Tests/PlayModeTests/SlateTests.cs

```diff
@@ -118,6 +118,32 @@ public IEnumerator Prefab_TouchZoom()
             yield return h.Hide();
         }
 
+        /// <summary>
+        /// Test ggv zooming instantiated from prefab
+        /// </summary>
+        [UnityTest]
+        public IEnumerator Prefab_GGVZoom()
+        {
+            InstantiateFromPrefab(Vector3.forward);
+
+            PlayModeTestUtilities.SetHandSimulationMode(HandSimulationMode.Gestures);
+
+            TestHand h = new TestHand(Handedness.Right);
```

Comment:
```suggestion
            TestHand rightHand = new TestHand(Handedness.Right);
```

Reply:
I think this was copied from some of the other code above/below, but in general we should avoid abbreviations unless they have some crazy high value.

https://google.github.io/styleguide/cppguide.html#General_Naming_Rules

Otherwise, you can end up in situations where you're reading something and see:

h.Show() - what is h? Is it handle? Is it hamburger? Is it hand? 
h2.Show() - what is h2? What is it's relationship to h? Are they ordinal? (no, because one is left, one is right, and there numbering actually doesn't matter)

Reply:
Compare that to:

leftHand.Show()
rightHand.Show()

And it's much clearer in the code what is happening.

In the context of this small function the value isn't as high, but as things get added to tests (which they will) and things change in the future (which they will), there's the chance that the distance increases to the point where the declaration of h isn't close enough for someone to be like "oh yeah it's the right hand"